src/lib/core/overlay/overlay-ref.ts

```diff
@@ -101,9 +103,8 @@ export class OverlayRef implements PortalHost {
     let backdropToDetach = this._backdropElement;
 
     if (backdropToDetach) {
-      backdropToDetach.classList.remove('md-overlay-backdrop-showing');
-      backdropToDetach.classList.remove(this._state.backdropClass);
-      backdropToDetach.addEventListener('transitionend', () => {
+      let styles = getComputedStyle(backdropToDetach);
```

Comment:
Instead of using `getComputedStyle`, let's just set `pointer-events: none` on the backdrop and dismiss it after whichever comes frist: transitionend or a short wait (say, 300ms). This will ensure that the backdrop doesn't get in the way of doing anything and that it always gets removed.